### PR TITLE
Gemify repo

### PR DESCRIPTION
**Pull request by rafaelgomesxyz**
Makes repo ready to be published as gem, with name `cpl`.

### CODE REVIEW

File: cpl.gemspec [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require_relative "lib/cpl/version"
+
+Gem::Specification.new do |spec|
+  spec.name    = "cpl"
+  spec.version = Cpl::VERSION
+  spec.authors = ["Justin Gordon", "Sergey Tarasov"]
+  spec.email   = ["justin@shakacode.com", "dzirtusss@gmail.com"]
```
Comment (justin808) [inline cpl.gemspec:9]: use sergey@shakacode.com

File: cpl.gemspec [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  spec.summary     = "Heroku to Control Plane"
+  spec.description = "Helper CLI for migrating from Heroku to Control Plane"
+  spec.homepage    = "https://github.com/shakacode/heroku-to-control-plane"
+  spec.license     = "MIT"
+
+  spec.required_ruby_version = ">= 2.7.0"
+
```
Comment (justin808) [inline cpl.gemspec:17]: Don't include specs or anything else unnecessary.

Markdown files? do other gems include them?

Comment (justin808) [reply cpl.gemspec:17]: https://github.com/shakacode/react_on_rails/blob/master/react_on_rails.gemspec#L19-L21

**Review by justin808 — CHANGES_REQUESTED** [review]

File: cpl.gemspec (hunk 1 above)
Comment (rafaelgomesxyz) [reply cpl.gemspec]: Done in https://github.com/shakacode/heroku-to-control-plane/pull/4/commits/49ea29b5be5a9cccad978f49db0ac3f1206054d7.

File: cpl.gemspec (hunk 2 above)
Comment (rafaelgomesxyz) [reply cpl.gemspec:17]: Done in a800686427262da3ba11fe7aa820991c4b60ab3e.

**Review by justin808 — APPROVED** [review]

**Pull request merged**